src/test/resources/deleteAllBundles.sh

```
@@ -19,6 +19,6 @@ then
 fi
 
 echo "Will delete all bundles on $url"
-source ../getToken.sh $url
+source ./getToken.sh $url
 curl -s -X DELETE "$url:2100/processes" -H "Authorization:Bearer $token"
```

Shellcheck

[warning] token is referenced but not assigned.

(SC2154)